Formula/skycoin-cx.rb

```diff
@@ -23,13 +23,15 @@
 class SkycoinCx < Formula
   desc "Skycoin CX is a general programming language based on affordances"
   homepage "https://github.com/skycoin/cx"
-  url "https://github.com/skycoin/cx/releases/download/v0.6.2/cx-0.6.2-bin-macos-x64.zip"
-  version "0.6.2"
-  sha256 "92c6badd27c675bd4b4c7a2db68a22cdf4b0ea3dfa12ea75c7322e3c47420b25"
+  url "https://github.com/skycoin/cx/releases/download/v0.7.0/cx-0.7.0-bin-macos-x64.zip"
+  version "0.7.0"
+  sha256 "20209b50054e453d87f738225f9d3830d89884366a7c391fdbbc5dfc3fe85ee8"
 
   bottle :unneeded
 
   def install
-    bin.install "cx"
+    ## v0.7.0 has a different folder structure in the zip package.
```

Comment:
really need to include this in formula ?

Reply:
Thx for the feedback. Will update now